src/runtime/contrib/arm_compute_lib/acl_runtime.cc

```diff
@@ -163,24 +149,61 @@ class ACLRuntime : public JSONRuntimeBase {
   struct CachedLayer {
     std::shared_ptr<arm_compute::IFunction> function;
     std::vector<arm_compute::Tensor> inputs;
-    std::vector<arm_compute::Tensor> const_inputs;
     std::vector<arm_compute::Tensor> outputs;
   };
 
+  /*!
+   * \brief Create an ACL tensor given the JSON representation.
```

Comment:
```suggestion
   * \brief Create an ACL tensor given the JSON representation. If scale and offset are given, then create a quantized ACL tensor.
```

- IMHO, this function can be one of the overloading functions of `MakeTensor` and moved to `acl_utils`.
- `MakeACLTensor` might be clearer.

Reply:
The issue with moving these to `acl_utils` is that `nodes_` and `data_entry_` from the json runtime are required. It might be more simple to remove utils and move the remaining functions into the runtime as static functions?

Reply:
Hmmm I see. Then this is different from other functions in `acl_utils`. We could use the different naming convention. Specifically, we could keep the function here and rename it to something like `MakeACLTensorFromDataEntry` (you might have a better name for it).

Reply:
Sounds good, thanks for the suggestion